polyfill/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "temporal",
+  "version": "1.0.0",
+  "description": "TC39 Temporal Proposal Polyfill",
+  "main": "index.js",
```

Comment:
As this is using ES modules; and rollup to generate a umd bundle, it'd be great to have a `module` keyword for build tools that can understand ES modules.